en/services/ftp.md

```diff
@@ -404,24 +404,57 @@ The GSC should create a timeout after the `RemoveDirectory` command is sent and
 
-kCmdOpenFileRO
-  - in _openAckResponse with opp kCOOpenBurst - change operation to kCOBurst
-  - new session! This is the burst session.
-  Set kCmdBurstReadFile 
-  
-  Ends when we hit EOF
-  kCmdBurstReadFile
-  
+<!-- Original Diagram
+sequenceDiagram;
+    participant Client
+    participant Server
+    Note right of Client: Open file for burst reading
+    Client->>Server:  BurstReadFile( data[0]=path, size=len(path) )
+    Server-- >>Client: ACK( session, size=4, data=len(file) )
+    Note left of Server: Server streams data in chunks with burst_complete=0
+    Server-- >>Client: BurstReadFile(session,burst_complete=0, offset=0, size=len(buffer), data[0]=buffer)
+    Server-- >>Client: BurstReadFile(session,burst_complete=0, offset=n, size=len(buffer), data[0]=buffer)
+    Note left of Server: When complete, send last chunk with burst_complete=1
+    Server-- >>Client: BurstReadFile(session,burst_complete=1, offset=last_chunk, size=len(buffer), data[0]=buffer)
+    Note right of Client: Read any missing chunks using ReadFile at offset
+    Client- >>Server:  ReadFile(session, size, offset)
+    Server-- >>Client: ACK(session, size=len(buffer), data[0]=buffer)
+    Note right of Client: Close session when whole data file collected
+    Client- >>Server:  TerminateSession(session)
+    Server-- >>Client: ACK()
 -->
 
+The sequence of operations is:
+1. Client (i.e. GCS) sends [BurstReadFile](#BurstReadFile) command specifying the file path to open.
+   - The payload must specify: `data[0]`= file path string, `size`=length of file path string.
+1. Server (drone) responds with either 
+   - ACK on success. The [payload](#payload) must specify fields: `session` = file session id, `size` = 4, `data` = length of file that has been opened. 
+   - NAK with [error information](#error_codes), e.g. `NoSessionsAvailable`, `FileExists`. 
+     The client may cancel the operation, depending on the error.
+1. Server sends stream of [BurstReadFile](#BurstReadFile) data to client (without ack) until whole file sent, or a server-defined burst size limit is reached.
```

Comment:
So essentially if you reach this limit then the steaming will stop with burst_complete=0 - the client will be waiting on timeout to proceed.